projects/uitdatabank/reference/entry.json

```
@@ -731,6 +731,69 @@
         "x-internal": true
       }
     },
+    "/events/{eventId}/attendance-mode": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        }
+      ],
+      "put": {
+        "summary": "attendance-mode - update",
```

```suggestion
        "summary": "attendanceMode - update",
```

For other operations I've also always used the property name instead of the path name. The reasoning being that an integrator would look how to update a specific property. The path names just happen to use dashes instead of camelCase because that's the convention for URLs.